src/components/kytos/inputs/Input.vue

```diff
@@ -21,11 +21,14 @@ import KytosBaseWithIcon from '../base/KytosBaseWithIcon';
 export default {
   name: 'k-input',
   mixins: [KytosBaseWithIcon],
+  compatConfig: {
+    MODE: 3,
```

Comment:
very nice that you changed the compatibility mode to 3, @HeriLFIU !

Reply:
can we change it globally ? is there any broken code that would be impacted ?

Reply:
Thank you, but I only changed it for two individual components, since the changes worked in vue3 but did not work in vue2, since they were incompatible.

Reply:
I've changed it globally to MODE3 on `webpack/webpack.common.js` just for testing and it seems to work fine (despite additional warnings on compilation)